bin/cli.js

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+  [ '--karma-config <value>',    'Karma config file'],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+for (const level in karmaLogLevels) {
+  karmaLogLevels[level][0] = karma.constants[karmaLogLevels[level][0]];
+}
+
+const isLogAtLeast = (level1, level2) => {
+  const value1 = logLevels.indexOf(level1 || 'disable');
+  const value2 = logLevels.indexOf(level2 || 'disable');
+  if (value1 === -1) throw new Error(`Invalid log level specified ${level1}`);
+  if (value2 === -1) throw new Error(`Invalid log level specified ${level2}`);
+  return value1 >= value2;
+};
+
+const merge = (arr1 = [], arr2 = []) => common.merge(arr1, arr2);
+
+const exclude = (files, filters) => filters.map(path => path.replace('.', '\\.')
+  .replace('*', '.+')
+  .replace('?', '.'))
+  .map(path => new RegExp(path))
+  .reduce((files, regexp) => files.filter(file => !regexp.test(file)), files);
+
+const parseFile = file => {
+  const data = fs.readFileSync(path.resolve(file), 'utf8');
+  switch (common.fileExt(file)) {
+    case 'json':
+      return JSON.parse(data);
+    case 'yml':
+    case 'yaml':
+      return yaml.parse(data);
+    default:
+      throw new Error('Unsupported config file format');
+  }
+};
+
+const dirname = dir => {
+  if (dir === '/') throw new Error('package.json does not exist');
```

Comment:
Also, you throw here which will supposedly lead to node exit. Then maybe we should just replace all 
```
console.error('File does not exist:', file);
process.exit(1);
```
and such with just `throw`?